lib/grppc.gi

```diff
@@ -272,7 +272,8 @@ InstallMethod( Pcgs,
 ##
 #M  GeneralizedPcgs( <G> )
 ##
-InstallImmediateMethod( GeneralizedPcgs, IsGroup and HasPcgs, 0, Pcgs );
+# is now set when creating
+#InstallImmediateMethod( GeneralizedPcgs, IsGroup and HasPcgs, 0, Pcgs );
```

Comment:
Well, it is set when creating via `GroupWithGenerators`, but that's not the only way to create a group object! However, I agree this does not need to be an immediate method; esp. as nothing tests `HasGroupWithGenerators`. So let's just turn it into a regular method.